pkg/pipeline/extract/conntrack/metrics.go

```diff
@@ -23,9 +23,16 @@ import (
 )
 
 var (
-	connStoreLengthDef = operational.DefineMetric(
-		"conntrack_memory_connections",
-		"The total number of tracked connections in memory.",
+	connStoreRunningLengthDef = operational.DefineMetric(
+		"conntrack_memory_connections_running",
+		"The total number of tracked running connections in memory.",
+		operational.TypeGauge,
+		"group",
+	)
+
+	connStoreExpiredLengthDef = operational.DefineMetric(
+		"conntrack_memory_connections_expired",
+		"The total number of tracked expired connections in memory.",
 		operational.TypeGauge,
 		"group",
```

Comment:
The labels are already `groupType.labelValue`
```
	cs.metrics.runningConnStoreLength.WithLabelValues(groupLabel).Set(float64(groupLen))
```

Are you suggesting to add a prefix like `running-` & `expired-` ? 
I'm fine with that (just need to confirm which prefixes :smile: )

Reply:
what I meant is, instead of having:
```
conntrack_memory_connections_running{group="0: DEFAULT"} 123
conntrack_memory_connections_expired{group="0: DEFAULT"} 456
```

having:
```
conntrack_memory_connections{group="0: DEFAULT", phase="running"} 123
conntrack_memory_connections{group="0: DEFAULT", phase="expired"} 456
```

Reply:
Let's move to this. Thanks @ronensc !